source/frontend/src/features/properties/map/acquisition/ViewSelector.tsx

```diff
@@ -25,6 +27,16 @@ export const ViewSelector = React.forwardRef<FormikProps<any>, IViewSelectorProp
     // render edit forms
     if (props.isEditing && !!props.acquisitionFile) {
       switch (props.activeEditForm) {
+        case EditFormNames.acquisitionChecklist:
+          return (
+            <UpdateAcquisitionChecklistContainer
+              formikRef={formikRef as any}
```

Comment:
Manuel would want me to ask why type coercion is necessary here.

Reply:
I'd have to check, I think because there are several forms that can receive the formik-ref but will see if typing can be improved